requirements.txt

```
@@ -1,5 +1,4 @@
 ratelim
-pyCrypto
-earthengine-api
+earthengine-api==0.1.335
```

```suggestion
earthengine-api
```
I would not pin this version. Or did you mean `>=`?

---

Fixed thanks